src/middleware/github-webhook-middleware.test.ts

```diff
@@ -53,6 +53,10 @@ describe("Probot event middleware", () => {
 			const spy = jest.fn();
 			await expect(GithubWebhookMiddleware(spy)(context)).toResolve();
 			expect(spy).toHaveBeenCalledTimes(3);
+
+			const anotherSpy = jest.fn();
+			await expect(GithubWebhookMiddleware(anotherSpy)(undefined, context)).toResolve();
+			expect(anotherSpy).toHaveBeenCalledTimes(3);
 		});
 
 		it("when one call fails", async () => {
```

Comment:
Can we change this to 'should <what we expect to happen> when one call fails'? Just makes it easier to understand what we're expecting from the test.